packages/desktop-client/src/components/reports/reports/NetWorth.jsx

```diff
@@ -150,13 +161,12 @@ export function NetWorth() {
           </Paragraph>
           <Paragraph>
             Net worth shows the balance of all accounts over time, including all
-            of your investments. Your “net worth” is considered to be the amount
-            you’d have if you sold all your assets and paid off as much debt as
-            possible. If you hover over the graph, you can also see the amount
-            of assets and debt individually.
+            of your investments. Your “net worth” is considered to be the debt
+            as possible. If you hover over the graph, you can also see the
+            amount of assets and debt individually.
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏oops? Accidentally removed some text here I think

Reply:
Oops. Didn't notice auto fix deleted them.